__tests__/app.test.js

```diff
@@ -65,6 +87,40 @@ describe('/api/articles', () => {
             })
         })
     })
+    test('GET:200 sends an array of articles to client filtered by topic', () => {
+        return request(app).get('/api/articles?filter=cats')
+        .expect(200)

```

Comment:
The task asked us to accept a topic query, so this should ideally be topic rather than filter.